big_tests/tests/metrics_api_SUITE.erl

```diff
@@ -312,3 +334,14 @@ assert_counter_inc(_Name, Inc, Counter1, Counter2) when Counter1 + Inc =:= Count
 
 get_diffs(L1, L2) ->
     lists:zip(L1, L2).
+
+ensure_nodes_not_clustered(Config) ->
+    Nodes = mnesia:system_info(running_db_nodes),
```

Comment:
I think we should not assume that Mnesia will be running in the test environment - it's a separate VM than the servers, why would it have to be running?

I'd rather fetch this information from `test.config`, using something similar to:

```erlang
Hosts = [ {Name, maps:from_list(Spec)}
          || {Name, Spec} <- distributed_helper:get_or_fail(hosts),
             {cluster, mim} <- [lists:keyfind(cluster, 1, Spec)] ],
ct:pal("hosts: ~p", [Hosts]).
```

Reply:
@erszcz You're, of course, right. What do you think about running `mnesia:system_info/1` but via rpc on one of mim nodes?

Reply:
> My intention here was to save the current state of MIM cluster, remove all nodes from the cluster for the sake of this particular test case and return to the same cluster state once the test case is finished.
Definitely makes sense :+1: